spec/wings/model_transformer_spec.rb

```diff
@@ -184,6 +184,72 @@
         expect(subject.build.lease.id.id).to eq work.lease.id
       end
     end
+
+    context 'with files and derivatives in fileset' do
```

Comment:
'with files and derivatives in fileset' is new to test that files on a fileset are correctly translated.